src/Button/variants.scss

```diff
@@ -89,7 +99,7 @@
   font-size: var(--font-size-l) !important;
 
   .nds-button-content {
-    margin: rem(10px) 0;
```

Comment:
Peter and I had the same fix - need to update from main and this diff line will go away